printer_mods/Printopal/Magnetic_top_panel/readme.md

```diff
@@ -0,0 +1,45 @@
+# Magnetic top panel
+# Overview
+This top panel can be easily removed for printer maintenance or giving more cooling.
+3D models are designed to match your buildplate size.
+For versions 1.8 and 2.4 in 250, 300 and 350mm.
+
+https://user-images.githubusercontent.com/85416803/120906040-da129680-c61b-11eb-813c-af05dc87f28a.mp4
+
+# BOM
+250:  
+400 x 400 x 3mm acrylic panel sheet  
+4x 344mm of 6 x 11,7mm [1/2” x 1/4”] High density foam
+
+300:  
+450 x 450 x 3mm acrylic panel sheet  
+4x 394mm of 6 x 11,7mm [1/2” x 1/4”] High density foam
+
+350:  
+500 x 500 x 3mm acrylic panel sheet  
+4x 444mm of 6 x 11,7mm [1/2” x 1/4”] High density foam
+
+Core:  
+8x Ø6 x 3mm neodymium magnet
+
+Acrylic sheet with abrasion resistant coating may be considered for V1.8 to prevent micro scratch caused by the reverse bowden tube. This may also appear with full height prints on V2.4.
+I don’t recommend using glass plate because of its weight, this may result in broken top skirt and broken glass!
+
+
+# Instructions
+1. Insert magnets into panel corner and frame corner respecting their polarity.
+2. Insert frame corner on top of the 4 vertical 2020 extrusions.
+3. Put panel corner on frame corner, sides should be flush.
+4. Glue the acrylic panel sheet on panel corners, be sure to keep panel corner and frame corner sides flush while gluing the acrylic panel.
```

Comment:
What kind of glue?  Or would VHB tape work?  Same questions for steps 6 and 7.

Reply:
Standard liquid superglue is used here.
WHB will add an extra heigth that will cancel the foam seal.

Reply:
Can you add that note to the README to make sure someone won't try to use VHB tape?

Reply:
Sure, it's updated